Source/MonoGame.Extended/Maps/Tiled/TiledObject.cs

```diff
@@ -1,18 +1,26 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using Microsoft.Xna.Framework;
 using MonoGame.Extended.Shapes;
 
 namespace MonoGame.Extended.Maps.Tiled
 {
     public class TiledObject
     {
```

Comment:
So something like this?

		public TiledObject(TiledObjectType objectType,int id,int? gid,float x,float y,float width,float height) : this(objectType,id,gid,new Vector2(x,y),width,height)
		{

		}

		public TiledObject(TiledObjectType objectType,int id,int? gid,Vector2 position,float width,float height)
		{
			ObjectType = objectType;
			Id = id;
			Gid = gid;
			Width = width;
			Height = height;
			Points = new List<Vector2>();
			Properties = new TiledProperties();
			Position = position;
		}

I did not think of creating the `Vector2` in `this()` , sorry about that.